### PR TITLE
✨ added ability to pass in gql middleware

### PR DESCRIPTION
**Pull request by jgnieuwhof**
adds ability to pass in gql middleware from parent app

### CODE REVIEW

**Comment by jgnieuwhof** [comment]
@alex-wilmer - want to get your opinion on this. netlify was having real issues with babel version inconsistencies, so I locked all module's babel versions to beta 40, which solved the problem, but now I'm worried about so many package-lock changes. I don't know, is it something to worry about? Is there a different sol'n or is this alright?

File: modules/schema/src/index.js [hunk 1]
```diff
@@ -71,5 +74,5 @@ export default ({
     });
   }
 
-  return schema;
+  return middleware?.length ? applyMiddleware(schema, ...middleware) : schema;
```
Comment (hlminh2000) [inline modules/schema/src/index.js]: tiny thing but does it fail with an empty array? if not we can probably remove this check, `middleware` is defaulted to an array anyway

Comment (jgnieuwhof) [reply modules/schema/src/index.js]: lol you're right, just tested w/ empty array, will update

**Comment by jgnieuwhof** [comment]
@hlminh2000 - addressed :)

**Pull request merged**